src/less/stage-jumbotron.less

```diff
@@ -60,10 +60,10 @@
 
 		.stage-search-btn {
 			height: 60px;
-			background-color: @white;
+			//background-color: @white;
```

Comment:
It is helpful to give commented out code a note, otherwise delete it as this becomes tech debt.

Reply:
Removed.